Demo/HWIFileDownload/DemoDownloadStore.m

```diff
@@ -395,17 +395,10 @@ - (nullable NSProgress *)rootProgress
     }
 }
 
-/*
 - (void)customizeBackgroundSessionConfiguration:(nonnull NSURLSessionConfiguration *)aBackgroundSessionConfiguration
 {
-    NSMutableDictionary *aHTTPAdditionalHeadersDict = [aBackgroundSessionConfiguration.HTTPAdditionalHeaders mutableCopy];
-    if (aHTTPAdditionalHeadersDict == nil) {
-        aHTTPAdditionalHeadersDict = [[NSMutableDictionary alloc] init];
-    }
-    [aHTTPAdditionalHeadersDict setObject:@"identity" forKey:@"Accept-Encoding"];
```

Comment:
You deleted these lines. It is not necessary anymore to send the "identity" value in the "Accept-Encoding" header? I remember adding these lines in the context of these discussions (https://stackoverflow.com/questions/36581495/how-to-get-nsurlsession-to-return-content-lengthhttp-header-from-server-objec).

Reply:
@Heikowi Wasn't aware of that, sorry. I just saw a commented code and I thought it was some sort of old code that wasn't needed anymore. I will revert that.

Reply:
Reverted 3ff0804.